enhancements/olm/olm-managed-operator-metrics.md

```diff
@@ -110,26 +106,58 @@ A `ServiceMonitor` object must be created in the namespace mentioned above and p
 
 ##### RBAC Requirements
 
-A role/rolebinding must be created that provides the Prometheus Operator ServiceAccount with the appropriate RBAC privileges to discover the `ServiceMonitor` and `PrometheusRule` resources in the newly created namespace.
+A role and rolebinding must be created that provides the OpenShift Monitoring `prometheus-k8s` ServiceAccount with the appropriate RBAC privileges to discover the `Services` in your operator's namespace. An example of the required RBAC can be seen below:
```

Comment:
Does this mean that we would have to package this RBAC or that OLM will create this RBAC for us?